_posts/2020-01-28-Tips-for-creating-open-source-projects-that-are-contributor-friendly.markdown

```diff
@@ -0,0 +1,57 @@
+date: 2020-01-28 06:31:01
+categories: [open source]
+author: bronzdoc
+
+---
+
+At Ombu Labs we are firm believers of open source. We use open source but also like to contribute back to the community and we have open sourced a few projects.
+
+We know that contributing to open source can be a difficult task so we have been looking for easier ways help to onboard new and past contributors.
+Here is a list of tips and tools that we believe that can help you make your open source project a more friendly space.
+
+# Documentation
+
+Having proper documentation for your project is key to make your project more friendly. Document the steps to setup the project, known issues and a guide on how to properly contribute to the project with a CONTRIBUTING.md or Adding steps in your README.md.
+
+# Label issues
+
+A nice thing that about issue trackers these days is that you can label issues with a short text to know what is about before reading the issue. With this a contributor can filter issues by its label and pick something to work on of his interest.
+
+Some nice labels for new contributors are:
+
+ * Help Wanted
+ * first-timers-only
+ * Beginner
+ * Good First Issue
+ * Easy
+
+
+# Code Of Conduct
+
+Having a code of conduct is important. Contributing to open source is more than code, there is people involved and a COC could help people feel safe and welcome by knowing there is a set of rules to maintain armony.
+
+We personally have been using [The Contributor Covenant](https://www.contributor-covenant.org/) but there are many more that you can change and adpat to your project needs.
+
+# Welcome New Contributors
+
+New contributions are always appreciated not matter if they are a feature, bug fix or an issue report. Some times this contributions could go unoticed and new contributors might feel discouraged and dissapointed for the lack of response. Luckily there are apps like [Welcome Github app](https://github.com/apps/welcome) that can help you with that.
+
+The welcome bot will post a message when a new issue is created, a PR is opened anf a PR is merged for a new contributor. This is a nice way to make a person contributing for the first time to your project welcome.
+
+This is an example of the welcome bot for our Skunk project
+[skunk welcome bot](https://i.imgur.com/JyqBXcs.png)
+
+# Code Reviews
+After a PR is submitted you would want to review it. Be objective but celebrate and encourage changes whever you can, this will not only make the person changes welcome but it will encourage them to keep working on the PR if you request some changes.
+
+# Communication
+Having a direct channel of communication with the users of your projects is a nice way to make them feel welcome. Depending on how big your projects is you could have a slack or an IRC channel for people wanting to contribute could directly ask open ended questions about howto solve a simple problem or any doubt they might have.
```

Comment:
@bronzdoc can you break this sentence? it's a bit hard to read it since it's too long